locale/en/LC_MESSAGES/django.po

```diff
@@ -4935,17 +4927,16 @@ msgid "Vergoeding"
 msgstr "Compensation"
 
 #: studies/templates/studies/study_form.html:110
-#, python-format
 msgid ""
 "\n"
 "                        Let op! Je hebt de vraag <a href=\"%(url)s\">'In "
-"welke hoedanigheid ben je betrokken bij deze studie?'</a> nog niet ingevuld, "
-"waardoor deze pagina nog kan veranderen op basis van je antwoord.\n"
+"welke hoedanigheid ben je betrokken bij deze aanvraag?'</a> nog niet "
+"ingevuld, waardoor deze pagina nog kan veranderen op basis van je antwoord.\n"
 "                    "
 msgstr ""
 "\n"
 "                Warning! The question <a href=\"%(url)s\">'In welke "
-"hoedanigheid ben je betrokken bij deze studie?'</a> has not been answered "
+"hoedanigheid ben je betrokken bij deze aanvraag?'</a> has not been answered "
 "yet. This page might change depending on your answer.\n"
 "            "
```

Comment:
Ehm, we might want to translate that question? :P
Het is ook in het engels als je naar die vraag springt, dus dit is gewoonweg raar.

Reply:
Het idee van recursieve vertalingen doet me een beetje zeer aan mijn brein. Maar misschien is het met de placeholder strings wel een mogelijkheid?